DataCollection/src/Domain/Managers/BirdSpeciesManager.cs

```diff
@@ -33,9 +35,12 @@ public BirdSpeciesDomainManager(IBirdSpeciesPersistence birdSpeciesPersistence)
 
         public IEnumerable<BirdSpecies> GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
         {
-            var retVal = BirdSpeciesPersistence.GetBirdSpeciesBySurveyTypeId(surveyTypeId);
-            return retVal;
-        }
+            var ctx = DataCollectionContext.Create();
+            var returnVal = new List<BirdSpecies>();
+            var surveyType = ctx.SurveyTypes.Include("BirdSpecies").FirstOrDefault(surveyItem => surveyItem.Id == surveyTypeId);
```

Comment:
If `IBirdSpeciesPersistence` doesn't have a

```
ICrudSet<SurveyType> SurveyTypes { get; }
```

Then you should add it, and at the top of `DataCollectionContext`, you can explicitly implement the interface:

```
ICrudSet<SurveyType> IBirdSpeciesPersistence.SurveyTypes { 
  get { return new CrudSetDecorator(this.SurveyTypes); }
}
```